mu-plugins/blocks/navigation/index.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * Generates a navigation block with hardcoded menu items, from a defined set of menus provided via a filter.
```

Comment:
```suggestion
 * Extends the core navigation block to use hardcoded menu items, from a defined set of menus provided via a filter.
```

Just a suggestion to make it very clear it's not a new block.